operator/pkg/client/users/client_test.go

```diff
@@ -32,6 +32,8 @@ import (
 	"github.com/redpanda-data/redpanda-operator/operator/internal/testutils"
 )
 
+const redpandaTestContainerImage = "docker.redpanda.com/redpandadata/redpanda:v23.2.8"
```

Comment:
There's an environment variable `TEST_REDPANDA_VERSION` in our flake.nix. Can you update this to pull the image version from there?

Reply:
Sure done.